randovania/games/dread/json_data/Artaria.txt

```diff
@@ -5638,8 +5164,6 @@ Extra - asset_id: collision_camera_086
       Can Slide
```

Comment:
Can probably wait til later, but can add an event for the speed block being destoyed + shoot missile, or an event for being in the bonus room + speed + shoot missile. Also depends if Slide even ever gets shuffled again.

Reply:
I'll wait until later